bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -33,165 +33,167 @@ Guntamatic Heating Systems supported as Thing Types:
+| `016-g1-target`            | G1 Target              | `Number:Dimensionless` | `%`  | 🔓 W0                 | R/O      | false    |
+| `017-buffer-top`           | Buffer Top             | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `018-buffer-mid`           | Buffer Mid             | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `019-buffer-btm`           | Buffer Btm             | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `020-pump-hp0`             | Pump HP0               | `Switch`               |      | 🔓 W0                 | R/O      | false    |
+| `021-dhw-0`                | DHW 0                  | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `022-b-dhw-0`              | B DHW 0                | `Switch`               |      | 🔓 W0                 | R/O      | false    |
+| `023-dhw-1`                | DHW 1                  | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `024-b-dhw-1`              | B DHW 1                | `Switch`               |      | 🔓 W0                 | R/O      | false    |
+| `025-dhw-2`                | DHW 2                  | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `026-b-dhw-2`              | B DHW 2                | `Switch`               |      | 🔓 W0                 | R/O      | false    |
+| `027-room-temp-hc-0`       | Room Temp:HC 0         | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `028-heat-circ-0`          | Heat Circ. 0           | `Switch`               |      | 🔓 W0                 | R/O      | false    |
+| `029-room-temp-hc-1`       | Room Temp:HC 1         | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `030-flow-target-1`        | Flow Target 1          | `Number:Temperature`   | `°C` | 🔐 W1                 | R/O      | false    |
+| `031-flow-is-1`            | Flow is 1              | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `032-mixer-1`              | Mixer 1                | `String`               |      | 🔐 W1                 | R/O      | false    |
+| `033-heat-circ-1`          | Heat Circ. 1           | `Switch`               |      | 🔐 W1                 | R/O      | false    |
+| `034-room-temp-hc-2`       | Room Temp:HC 2         | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `035-flow-target-2`        | Flow Target 2          | `Number:Temperature`   | `°C` | 🔐 W1                 | R/O      | false    |
+| `036-flow-is-2`            | Flow is 2              | `Number:Temperature`   | `°C` | 🔓 W0                 | R/O      | false    |
+| `037-mixer-2`              | Mixer 2                | `String`               |      | 🔐 W1                 | R/O      | false    |
```

Comment:
Not sure if it is feasable,did you consider to create (dynamic) groups as there seem to be 8 sets of circuits with their own channels for temp, flow target etc etc). 

Maybe similar for the dhw heating?

Reply:
As the data provided by the heating system are fully dynamic I would like to avoid the implementation of any logic here. This will only create further issues, as every heating system, firmware, ... behaves different. See also the comment above.

Reply:
Fair enough